### DISCUSSION TITLE
Search and create should order notes with title match first

### DISCUSSION THREAD

**Discussion by gal064** [glushchenko/fsnotes#1655] [Ideas]
Pretty simple. When I search for a note, I want to first see notes that their title match my search and only then, notes where the content match my search

**Comment by gingerbeardman** [comment]
I've opened a "feature request" issue for this as #1656

**Comment by glushchenko** [comment]
> Pretty simple. When I search for a note, I want to first see notes that their title match my search and only then, notes where the content match my search

Please check FSNotes 6.6.3

**Comment by gal064** [comment]
Amazing thanks! Great product
